public/app/features/dashboard/utils/publicDashboards.ts

```diff
@@ -0,0 +1,5 @@
+import { getConfig } from '../../../core/config';
+
+export const isPublicDashboardView = () => {
+  return getConfig().featureToggles.publicDashboards && window.location.pathname.split('/')[1] === 'p';
+};
```

Comment:
Grafana supports being served on a subpath (so the homepage is at `example.com/foo`, and public dashboards would be at `example.com/foo/p/`), and this doesn't look like it would support that.

We have some utils in packages/grafana-data/src/utils/location to help with this :)

Reply:
Nice, did not know about that one